### PR TITLE
Updated help command

### LINKED ISSUE #105: Improve cli documentation on README and in the help cli command
As has been indicated in this issue #104, it would be great to update the readme and help command in order to make the usage of the cli clear.

It can be use examples, like the one found on the #104

### PR DESCRIPTION
**Pull request by jlc893**
# Pull Request

## Description

Updates the help command to improve readability and usability. I reworded the help command to avoid confusions such as #104 . I don't believe any changes need to be made to the README file. This is my first real open source contribution so please let me know if I did anything wrong.

Fixes #105 

## Type of change

- [ ]  This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* IGN Version:

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/cli.py
Comment (davideimola) [inline src/cli.py:26]: I think it can sound better, but I accept other suggestions here :smile: 

```suggestion
                                    only nord), by replacing 'PALETTE' with a
                                    theme name it is possible to select it. If
```

**Review by davideimola — COMMENTED** [review]
Hi @jlc893! Great work on your first-time contribution! I am really happy that you have decided to take care of something like that in one of our projects!

I leave you a small suggestion, I have changed it because it sounds better but I am open to other possible suggestions.

Until now, great work and keep going like that!

File: src/cli.py
Comment (jlc893) [reply src/cli.py]: I would agree that sounds better. I have made another commit with that change. Is it good to be merged now? Thanks for the feedback!

**Comment by jlc893** [comment]
> Hi @jlc893! Great work on your first-time contribution! I am really happy that you have decided to take care of something like that in one of our projects!
> 
> I leave you a small suggestion, I have changed it because it sounds better but I am open to other possible suggestions.
> 
> Until now, great work and keep going like that!

**Review by davideimola — APPROVED** [review]
LGTM, waiting for @Wabri to make a review and after that we can merge it!

File: src/cli.py
Comment (Wabri) [inline src/cli.py:32]:
```suggestion
                                    To use the Monokai theme use this command: python src/cli.py --monokai
```

**Review by Wabri — CHANGES_REQUESTED** [review]
Thank you for this or 😎 this little change and we can merge it 👍

**Comment by jlc893** [comment]
I just made your suggested change. Thanks!

**Review by Wabri — APPROVED** [review]

**Pull request merged**
